bin/morgue.js

```diff
@@ -7437,6 +7438,136 @@ async function metricsImporterCmd(argv, config) {
   await cli.routeMethod(argv);
 }
 
+function stabilityCreateMetric(coroner, argv, config) {
+  let universe = argv.universe;
+  if (!universe) {
+    universe = Object.keys(config.config.universes)[0];
+  }
+
+  let project = argv.project;
+
+  if (!universe) {
+    errx("--universe is required");
+  }
+
+  if (!project) {
+    errx("--project is required");
+  }
+
+  let bpg = coronerBpgSetup(coroner, argv);
+  let model = bpg.get();
+
+  /* Find universe. */
+  let uid = 0;
+  for (let u of model.universe) {
+    if (u.get("name") === universe) {
+      uid = u.get("id");
+    }
+  }
+
+  if (uid == 0) {
+    errx("Universe not found");
+  }
+
+  let pid = 0;
+  for (let p of model.project) {
+    if (p.get("universe") === uid && p.get("name") == project) {
+      pid = p.get("pid");
+    }
+  }
+
+  if (pid === 0) {
+    errx("Project not found");
+  }
+
+  const metricGroupName = argv["metric-group"];
```

Comment:
We should do validation at the beginning of the function. Let's imagine a situation user didn't specify this argument. We can capture this situation much faster without unnecessary code execution.

Reply:
I've moved all the validation up.